### ISSUE TITLE
Storelocator dependency

### ISSUE THREAD

**Issue by lukaszdobrowolskithecoded** [Magestore/simple-magento2-admin-module#1]
Your module depends on Magestore\Storelocator module that is not included.
Without it it is impossible to run module

**Comment by NagarajuKasa** [comment]
Yes While compiling the module i am getting error  as shown in this screenshot 
http://prntscr.com/gprmnp

Please help me to fix this error. Thanks i hope you will  provide update on this please help me @lukaszdobrowolskithecoded

**Comment by eduardo-g-silva** [comment]
just delete the helper, Data.php and will work fine

**Comment by shoaibgits** [comment]
> 
> 
> just delete the helper, Data.php and will work fine

And the helper is not required in further process? or you mean only delete it for the compilation and add it after?
